public/app/features/dashboard-scene/scene/RowRepeaterBehavior.ts

```diff
@@ -180,8 +180,12 @@ function updateLayout(layout: SceneGridLayout, rows: SceneGridRow[], maxYOfRows:
     const diff = maxYOfRows - firstChildAfterY;
 
     for (const child of childrenAfter) {
-      if (child.state.y! < maxYOfRows) {
```

Comment:
Not sure if I'm missing something but I think we don't need this if here. There are cases where the child state y is larger than the max y of repeated rows and we do want to offset those as well.